dbt_copilot_helper/templates/pipelines/environments/buildspec.yml

```diff
@@ -3,7 +3,7 @@
 version: 0.2
 env:
   variables:
-    COPILOT_VERSION: 1.31.0
+    COPILOT_VERSION: 1.32.0
```

Comment:
Are we keeping this up to date in terms of minor versions only? The latest version is `1.32.1` otherwise.

Reply:
This matches the ci-image-builder version currently in use. We do need to come up with a better strategy for this and ensure we are all running the same version.

Reply:
That makes sense. Would be easier if we merge the `ci-image-builder` into this repo!